projects/plugins/jetpack/extensions/shared/components/stripe-nudge/index.jsx

```diff
@@ -11,6 +16,7 @@ export const StripeNudge = ( { blockName } ) => {
 	const store = select( membershipProductsStore );
 	const stripeConnectUrl = store.getConnectUrl();
 	const { tracks } = useAnalytics();
+	const isWpcom = isAtomicSite() || isSimpleSite();
```

Comment:
`isWpcom` would be a nice little helper for `@automattic/jetpack-shared-extension-utils` as well.